crypto/o_fopen.c

```diff
@@ -44,7 +47,7 @@ FILE *openssl_fopen(const char *filename, const char *mode)
         if (MultiByteToWideChar(CP_UTF8, flags,
                                 filename, len_0, wfilename, sz) &&
             MultiByteToWideChar(CP_UTF8, 0, mode, strlen(mode) + 1,
-                                wmode, OSSL_NELEM(wmode)) &&
+                                wmode, sizeof(wmode) / sizeof(wmode[0])) &&
```

Comment:
Just in case if someone wonders. I said many times that I don't appreciate project-specific lingo. It's appropriate if it saves space, but for occasional use like here, it's really not worth the trouble (of creating dependency on yet another header).

Reply:
Frankly, while I hear your opinion (and disagree), I think it's bad form to picky-back that particular argument on this PR.

Reply:
As I tried to minimize amount of headers, OSSL_NELEM disappeared. So it's wasn't like just for no reason :-) Well, now I had to put internal/cryptlib.h back, so it's back again. Should I put OSSL_NELEM back?

Reply:
I would rather see that you did, yes

Reply:
Just in case for reference. Rationale is that just like you wouldn't appreciate to end up in conversation when everybody is throwing around unfamiliar abbreviations, reader wouldn't directly appreciate unfamiliar macros. Yes, one can argue that macro can be intuitive (sure that it would directly be for non-English speaker? ;-), but consider auditor who has to know for sure... If there is one occurrence of and idiom, or even a couple, there is really no reason to replace it with lingo... Reverted.